src/App.js

```diff
@@ -13,8 +13,10 @@ import { Auth0Provider } from "./auth/react-auth0-wrapper";
 import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom'
 
 import DataSetListViewWrapper from './pages/dataset-list-view'
-import DatasetView from './pages/dataset-detail-view'
+import DatasetView from './pages/dataset-view'
+import DatasetDetailsView from './pages/dataset-detail-view'
```

Comment:
Is there any concern with the "Details" being plural in the import but singular in the path name?

Reply:
Yeah we can change that